pkg/tsdb/cloudmonitoring/time_series_filter.go

```diff
@@ -268,27 +268,34 @@ func (timeSeriesFilter *cloudMonitoringTimeSeriesFilter) handleNonDistributionSe
 	setDisplayNameAsFieldName(dataField)
 }
 
-func (timeSeriesFilter *cloudMonitoringTimeSeriesFilter) parseToAnnotations(queryRes *tsdb.QueryResult, data cloudMonitoringResponse, title string, text string, tags string) error {
-	annotations := make([]map[string]string, 0)
-
-	for _, series := range data.TimeSeries {
-		// reverse the order to be ascending
+func (timeSeriesFilter *cloudMonitoringTimeSeriesFilter) parseToAnnotations(queryRes *tsdb.QueryResult, response cloudMonitoringResponse, title string, text string, tags string) error {
+	frames := data.Frames{}
+	for j := 0; j < len(response.TimeSeries); j++ {
```

Comment:
why not for-each loop the timeseries? I'm fine either way, but please swap the index variable names so that this first loop index is named `i` and the second loop index is named `j`

Reply:
Oh I created the inner look first and the outer after, so I forgot to change the names. Will check the for-each look also to see if it's better, thanks!